README.adoc

```diff
@@ -51,44 +50,44 @@ The plugin provides bootstrap script that creates sample project with:
 * proper source file layout
 * sample https://gatling.io/docs/current/general/simulation_structure/[Simulation class], demonstrating sufficient `Gatling` functionality
 
-For this quickstart - `git` and `curl` must be instlled and available in `$PATH`.
+For this quickstart: `git` must be installed and available in `$PATH`, and a JDK must be installed and `$JAVA_HOME` configured.
```

Comment:
I have some ideas about sample project repo. Let me check first and may be i would provide a PR to it. 

@Isammoc May you add me as a collaborator to that project ?

Reply:
Also, issues and wiki should be disabled for demo repo.